### PR TITLE
Typescript and organize raps

### PR DESCRIPTION
**Pull request by jinchung**
1. Move entities file into a directory structure to hold types that are shared across the codebase
2. Typescript raps/actions and add new types
3. Organize raps directory to utilize raps index file and remove use of subdirectories in raps

Testing:
* All raps still work as expected (unlocking, swapping, depositing to Compound, withdrawing from Compound)
* Approvals still work as expected

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -17,14 +17,17 @@
       "@rainbow-me/config": ["./src/config"],
       "@rainbow-me/entities": ["src/entities"],
       "@rainbow-me/handlers": ["./src/handlers"],
+      "@rainbow-me/handlers/*": ["./src/handlers/*"],
```
Comment (jinchung) [inline tsconfig.json:20]: Just a note: when in a `ts` file and trying to import subdirectories like `@rainbow-me/handlers/uniswap`, it was showing an error: `Cannot find module '@rainbow-me/handlers/uniswap' or its corresponding type declarations.` unless the `/*` path was included in `tsconfig.json`

File: src/navigation/index.js [hunk 2]
```diff
@@ -1,4 +1,3 @@
-/* eslint-disable import/no-cycle */
```
Comment (brunobar79) [inline src/navigation/index.js:1]: 🙌

**Review by brunobar79 — APPROVED** [review]
🔥 🔥 🔥

**Review by skylarbarrera — APPROVED** [review]
🌮

**Pull request merged**